.github/workflows/builder_docker-based_slsa3.yml

```diff
@@ -213,7 +212,7 @@ jobs:
           set -euo pipefail
 
           REF="@${GITHUB_REF}"
-          if [[ "${GITHUB_EVENT_NAME}" == "pull_request" ]]; then
+          if [[ "${GITHUB_EVENT_NAME}" == "pull_request" ]] || [[ "${GITHUB_EVENT_NAME}" == "merge_group" ]]; then
```

Comment:
I actually don't think I need this since AFAICT `merge_group` has appropriate permissions.